acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/signing/BlsSigningAcceptanceTest.java

```diff
@@ -147,8 +148,12 @@ public void failsIfSigningRootDoesNotMatchSigningData() throws JsonProcessingExc
     assertThat(response.getStatusCode()).isEqualTo(500);
   }
 
-  @Test
-  public void ableToSignWithoutSigningRootField() throws JsonProcessingException {
+  @ParameterizedTest
+  @EnumSource(
```

Comment:
Would also be good to have this on a test that sends through the entire set of data, say the signDataWithKeyLoadedFromUnencryptedFile AT